Service/Reorder/FormatThicknessService.cs

```diff
@@ -39,14 +39,14 @@ public void ProcessElement(XElement element)
             if (element.Name == SetterName)
             {
                 var propertyAttribute = element.Attributes("Property").FirstOrDefault();
-                if ((propertyAttribute != null)
-                    && this.ThicknessAttributeNames.Any(_ => _.IsMatch(propertyAttribute.Value)))
-                {
-                    var valueAttribute = element.Attributes("Value").FirstOrDefault();
-                    if (valueAttribute != null)
-                    {
-                        this.FormatAttribute(valueAttribute);
-                    }
+				if ((propertyAttribute != null)
+				    && this.ThicknessAttributeNames.Any(_ => _.IsMatch(AttributeHelper.GetName(propertyAttribute))))
+				{
+					var valueAttribute = element.Attributes("Value").FirstOrDefault();
+					if (valueAttribute != null)
+					{
+						this.FormatAttribute(valueAttribute);
+					}
```

Comment:
Replace tabs with spaces, which will reduce the number of changes here. You can change this setting in Tools --> Options --> Text Editor --> All Languages --> Tabs. In general, use 4 spaces instead of a tab.